SeaPublicWebsite.Data/User.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace SeaPublicWebsite.Data;
```

Comment:
Can we make this `UserData` rather than `User` please

Reply:
Done - i've left the dbset item in dbcontext named user as that still seemed the most sensible choice